src/commands/time.js

```diff
@@ -53,50 +66,43 @@ module.exports = {
 							'the set of attributes to send with this record',
 						type: 'array',
 					},
-					accountId: {
-						alias: 'id',
-						describe: 'The New Relic account ID',
-						demandOption: true,
-						default: process.env.NEW_RELIC_ACCOUNT_ID,
-					},
 					key: {
-						describe: 'The New Relic API insert key',
+						describe: 'The Datadog API key',
 						demandOption: true,
-						default: process.env.NEW_RELIC_INSERT_KEY,
+						default: process.env.DATADOG_API_KEY,
+					},
+					build: {
+						describe: 'The build number',
+						demandOption: false,
+						default: process.env.TRAVIS_BUILD_NUMBER,
 					},
 					appName: {
 						describe: 'The name of the app reporting data',
 						default: process.env.NEW_RELIC_APP_NAME,
 					},
 				},
 				argv => {
+					const NOW = new Date().getTime() / 1000; // one timestamp in seconds for all reported metrics
 					// create a record based on the requested attributes
 					const current = getCurrentTimes();
-					const record = argv.attributes.reduce(
-						(acc, attribute) => {
-							if (!current[attribute]) {
-								throw new Error(`${attribute} not started`);
-							}
-							if (!current[attribute].end) {
-								throw new Error(`${attribute} not finished`);
-							}
-							const currentTime =
-								current[attribute].end -
-								current[attribute].start;
-							acc[attribute] = currentTime / 1000; // report in seconds
-							return acc;
-						},
-						{ appName: argv.appName }
-					);
-
-					// initialize the New Relic Insights API
-					const track = new Insights({
-						insertKey: argv.key,
-						accountId: argv.accountId,
-						defaultEventType: argv.type,
+					const metrics = argv.attributes.map(attribute => {
```

Comment:
prefer `attr` here, because there is an `attribute` var declared in the parent scope, so better to [avoid shadowing](https://eslint.org/docs/rules/no-shadow). Alternatively, rename the parent scope `attribute` var to `attributeOptConfig` or something

(we should add that no-shadow rule to the eslint config, but no big need to do it now)